ReactiveCocoa/Swift/Signal.swift

```diff
@@ -223,8 +229,7 @@ extension SignalType {
 	}
 
 	/// Maps errors in the signal to a new error.
-	@warn_unused_result(message="Did you forget to call `observe` on the signal?")
```

Comment:
I would leave these in: I think this message IS useful because it explains what users are supposed to do with the resulting `Signal`.